### DISCUSSION TITLE
L12 testgetUsdValue fails , FAIL. Reason: EvmError: Revert

### DISCUSSION THREAD

**Discussion by Bhavik-punmiya** [Cyfrin/foundry-full-course-cu#867] [Q&A] [answered]
I am currently trying to run my first test but  I am running into these error I tried looking to existing discussion but none of that helped me solving these isssue 
I am getting these error : ``` Encountered 1 failing test in test/unit/DSCEngineTest.t.sol:DscEngineTest
[FAIL. Reason: EvmError: Revert] testGetUsdValue() (gas: 12633) ```

I am providing my [github-repository](https://github.com/Bhavik-punmiya/Decentralized-Stable-Coin), Please let me know If i am wrong some where 
![ssss](https://github.com/Cyfrin/foundry-full-course-f23/assets/118907757/c0d09f30-30e3-4706-94f6-7e9f369ef220)

**Comment by AztecWarlord** [comment, marked as answer]
In your ```setUp()``` you are not calling the ```ERC20Mock``` to mint weth tokens.
